x/staking/keeper/delegation.go

```diff
@@ -610,33 +609,21 @@ func (k Keeper) RemoveRedelegation(ctx context.Context, red types.Redelegation)
 // timeslice is a slice of DVVTriplets corresponding to redelegations that
 // expire at a certain time.
 func (k Keeper) GetRedelegationQueueTimeSlice(ctx context.Context, timestamp time.Time) (dvvTriplets []types.DVVTriplet, err error) {
-	store := k.storeService.OpenKVStore(ctx)
-	bz, err := store.Get(types.GetRedelegationTimeKey(timestamp))
+	triplets, err := k.RedelegationQueue.Get(ctx, timestamp)
 	if err != nil {
```

Comment:
let's do: `if err != nil && !errors.Is(err, collections.ErrNotFound) { return err }`; it's more readable.